ansible/my-ansible-playbook.yml

```
@@ -0,0 +1,91 @@
+- name: Simple Playbook
+  hosts: managed_nodes
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: secret_key
+      prompt: Input the SECRET KEY
+
+    - name: base_api_url
+      prompt: Input the TRELLO BASE API
+
+    - name: board_id
+      prompt: Input the Corndel BOARD ID
+
+    - name: todo_list_id
+      prompt: Input the Corndel TODO LIST ID
+
+    - name: done_list_id
+      prompt: Input the Corndel DONE LIST ID
+
+    - name: trello_api_key
+      prompt: Input the TRELLO API KEY
+
+    - name: trello_token
+      prompt: Input the TRELO TOKEN
+
+  tasks:
+  - name: Install Git on managed host
+    ansible.builtin.yum:      
+      name: git
+      state: installed
+    become: yes
+
+  - name: Install Python 3.11 on managed host
+    ansible.builtin.yum:
+      name: python311
+      state: installed
+    become: yes
```

Primarily a stylistic choice, but note that you can combine these:
```suggestion
  - name: Install Git & Python 3.11 on managed host
    ansible.builtin.yum:      
      name: 
        - git
        - python311
      state: installed
    become: yes
```